google-cloud-trace/lib/google/cloud/trace/patches/active_call_with_trace.rb

```diff
@@ -41,5 +41,8 @@ def request_response *args, **kwargs
   end
 
   # Patch GRPC::ActiveCall#request_response method
-  ::GRPC::ActiveCall.prepend ActiveCallWithTrace
+  # @api private
```

Comment:
You can also just say `@private` which is what we do elsewhere when we want to hide stuff from YARD. Then you wouldn't need the yardopts change.

Reply:
updated to use @private 👍 Thanks @dazuma